### DISCUSSION TITLE
Activity Matrix widget confusion

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#17383] [Mobile] [answered]
Change the Coloring from the activity-matrix-widget. 

My Problem:
When not contributed, the field is black
On Computer: The more contribution, the lighter the field gets
When on phone: The more contribution, the more saturated the field gets (but brightness is the same).

I would like to have it like on pc.

PC:
![grafik](https://user-images.githubusercontent.com/50182241/170702603-3fdcff34-4d7f-4233-8351-a8e32ab854b0.png)

Widget:
![BB8D4AC3-E449-4804-9E3B-BA198D18B598](https://user-images.githubusercontent.com/50182241/170702689-cef70b91-9c1c-4a89-a871-7a8e7f002c49.jpeg)

My Phone:
iPhone 8
iOS 15.6
Github Mobile 1.66.0

**Comment by brianlovin** [comment, marked as answer]
Thanks for reporting this @CuzImBisonratte — we've encountered this once before, and it appears to be a bug in iOS itself that is causing the colors in the widget to not properly sync with dark mode. Could you try giving your phone a restart to see if things look correct after?

**Reply by CuzImBisonratte** [reply]
I restarted it multiple times, and even then it doesn't change

**Reply by brianlovin** [reply]
Thanks for trying that @CuzImBisonratte — In this case, there might be something else going on. I'll flag this for our team 😊
